modules/livestore/kafka_client_franz.go

```diff
@@ -0,0 +1,71 @@
+package livestore
+
+import (
+	"context"
+
+	"github.com/go-kit/log"
+	"github.com/grafana/tempo/pkg/ingest"
+	"github.com/twmb/franz-go/pkg/kadm"
+	"github.com/twmb/franz-go/pkg/kgo"
+	"github.com/twmb/franz-go/plugin/kprom"
+)
+
+// FranzKafkaClient wraps a kgo.Client and admin client to implement the KafkaClient interface
+type FranzKafkaClient struct {
+	client  *kgo.Client
+	adminCl *kadm.Client
+}
+
+// NewFranzKafkaClient creates a new Franz/kgo-based Kafka client
+func NewFranzKafkaClient(cfg ingest.KafkaConfig, metrics *kprom.Metrics, logger log.Logger) (KafkaClient, error) {
```

Comment:
surprised we need this. aren't we already consuming traces from kafka in the block builder/metrics generator. do we need a new client?